app/javascript/react/components/DayView/DayView.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+import * as S from "./DayView.style";
+import { ThresholdsValues } from "../../utils/ThresholdsValues";
+
+interface DayViewProps {
+  value: number;
+  date: Date;
+  thresholdsValues: ThresholdsValues;
+}
+
+const DayView = ({ value, date, thresholdsValues }: DayViewProps) => {
+  return (
+    <S.Container>
+      <S.DesktopLabel>{value} (µg/m)</S.DesktopLabel>
+      <S.BackgroundBarContainer
+        value={value}
+        thresholdsValues={thresholdsValues}
+      >
```

Comment:
Please use transient props, as mentioned in Monika's PR, those props here are passed to React, to DOM elements and are not a native properties of div element, console complaining ;)